frontend/src/scenes/events/propertyDefinitionsLogic.ts

```diff
@@ -0,0 +1,67 @@
+import { kea } from 'kea'
+import api from 'lib/api'
+import { PropertyDefinition } from '~/types'
+import { propertyDefinitionsLogicType } from './propertyDefinitionsLogicType'
+
+interface PropertyDefinitionStorage {
+    count: number
+    next: null | string
+    results: PropertyDefinition[]
+}
+
+export const propertyDefinitionsLogic = kea<
```

Comment:
This is almost the same code as `eventDefinitionsLogic` (adjusted for types, and a few changed selectors), wondering if there's something worth refactoring in the spirit of DRY.

Reply:
I think it's fine. DRY is not a golden standard, but a suggestion.